dashboard/test/ui/features/pd/facilitatorApplicationDashboard.feature

```diff
@@ -0,0 +1,23 @@
+@dashboard_db_access
+@eyes
+
+Feature: Facilitator Application Dashboard view
+
+  Scenario: Application dashboard, list view, detail view, cohort view
+    Given I am a workshop administrator with some applications of each type and status
+    And I am on "http://studio.code.org/pd/application_dashboard/summary"
+    Then I wait until element "table" is visible
+    And I open my eyes to test "Facilitator Application Dashboard"
+    And I see no difference for "Application Dashboard"
+    Then I press the first ".row .btn" element
+    Then I wait until element "table" is visible
+    And I see no difference for "Facilitator List View"
+    Then execute JavaScript expression "window.location = $('table a:first-child').first().prop('href')"
```

Comment:
Worth adding a small comment here explaining why we're setting `window.location` directly (just documenting what you told me earlier).